### PR TITLE
Add configurable name so we don't only need "isFetching" -- (issue 10)

### PR DESCRIPTION
**Pull request by bxie1**
https://github.com/kitconcept/robotframework-react/issues/10

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -120,13 +120,17 @@ When working with Redux, you have to pass the name of the reducer to the 'Wait f
 
   Wait for react  reducer=headlines
 
-The reducer needs to implement an "isFetching" attribute in the Redux state::
+The reducer shouldrimplement an "isFetching" attribute in the Redux state::
```
Comment (tisto) [inline README.rst]: @bxie1 typo :)

**Comment by tisto** [comment]
@bxie1 thank you for your pull request! Would you mind adding your name and the change to our changelog? https://github.com/kitconcept/robotframework-react/blob/master/CHANGES.rst

**Comment by bxie1** [comment]
Closed the pull request by accident!

**Comment by jg123** [comment]
@tisto Anything this needs in order to be merged?

**Comment by tisto** [comment]
@jg123 no. Thanks for the reminder.

**Pull request merged**